3.2/root/usr/share/container-scripts/mongodb/mongodb.conf.template

```diff
@@ -6,13 +6,13 @@ port = ${CONTAINER_PORT}
 # Default: /var/lib/mongodb/data
 dbpath = ${MONGODB_DATADIR}
 
-# Disable data file preallocation. Default: true
+# Disable data file preallocation.
 # only for mounted data directory from older MongoDB server
-noprealloc = ${MONGODB_NOPREALLOC}
+noprealloc = true
```

Comment:
In the original comment @omron93 said that "Config file can contain it (it takes effect only if different storage engine is specified or data from old engine are present)."